category_theory/full_subcategory.lean

```diff
@@ -16,12 +16,14 @@ instance full_subcategory (Z : C → Prop) : category.{u v} {X : C // Z X} :=
   id   := λ X, 𝟙 X.1,
   comp := λ _ _ _ f g, f ≫ g }
 
-def full_subcategory_embedding (Z : C → Prop) : {X : C // Z X} ⥤ C :=
+def full_subcategory_inclusion (Z : C → Prop) : {X : C // Z X} ⥤ C :=
 { obj := λ X, X.1,
   map := λ _ _ f, f }
 
-instance full_subcategory_full     (Z : C → Prop) : full     (full_subcategory_embedding Z) := by obviously
-instance full_subcategory_faithful (Z : C → Prop) : faithful (full_subcategory_embedding Z) := by obviously
+instance full_subcategory_full     (Z : C → Prop) : full     (full_subcategory_inclusion Z) :=
```

Comment:
Shouldn't we just provide an instance of `fully_faithful`?

Reply:
Sure! (I think these predated the appearance of the `fully_faithful` typeclass).

Fixed.